src/app/pages/friend-list/pages/friend-list-details/friend-list-details-page.html

```diff
@@ -0,0 +1,22 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button defaultHref="/tabs/friend-list"></ion-back-button>
+    </ion-buttons>
+    <ion-title>Details</ion-title>
+    <ion-buttons slot="end">
+      <ion-button (click)="onDeleteFriend()">
```

Comment:
Is it necessary to use delete button in header? Maybe it should be moved inside content.